go.mod

```diff
@@ -1,18 +1,19 @@
 module github.com/ava-labs/precompile-evm
 
-go 1.22.8
+go 1.23.6
 
 require (
-	github.com/ava-labs/avalanchego v1.12.2
-	github.com/ava-labs/subnet-evm v0.7.1
+	github.com/ava-labs/avalanchego v1.13.0
```

Comment:
we also need to bump the version in .devcontainer.json.

Reply:
Steps added to #91